### ISSUE TITLE
Interval tool sound plays on unwanted interactions

### ISSUE THREAD

**Issue by matthew-blackman** [phetsims/center-and-variability#308]
In discussing https://github.com/phetsims/center-and-variability/issues/236, @catherinecarter noticed that the interval tool sound sometimes plays at unwanted times. We found that the tool sometimes plays a "double sound" on click/release, and possibly on very small mouse movements. We should fix this so that the interval tool sound only plays when the user is making intentional changes to the tool.

**Comment by matthew-blackman** [comment]
@catherinecarter the devs confirmed that the "double" sound was caused by an additional burst of sound playing when the mouse was moved very small distances (even 1 pixel). To fix this, I set the sound to play only if the interval tool has changed by a certain amount. 

As of the above commit, the interval sound is now triggered when either:
- The width of the interval tool has changed by a certain minimum amount (constant 1)
- The location of the interval tool has changed by a certain minimum amount (constant 2)

The threshold can be adjusted easily to be more or less sensitive. Let me know if you think there is promise here by adjusting the threshold values, and how you feel this sounds overall. 

Side note - Personally I think that adding the threshold makes the interval sounds better overall, since it feels more reactive to the user's actions instead of always playing  a sound on any change. Curious how you feel about this!

**Comment by catherinecarter** [comment]
I played with it a bit and the sound of the tool sounds really good. I agree, the sound feels more connected with my actions.

I think the tolerance is too high. I was able to wiggle the tool about 1/2 or more before the sound activated. When the sound came on, it was the the same sound even though the handle was in two different positions. 

Let's decrease the tolerance and see how it feels/sounds.
